README.md

```diff
@@ -1,17 +1,26 @@
 # Awesome Learning
 [![Build Status](https://travis-ci.org/wayfair/awesome-learning.svg?branch=master)](https://travis-ci.org/wayfair/awesome-learning)
 
-Learn JavaScript and Front-End Fundamentals at your own pace.
+Awesome Learning is a Frontend focused learning platform created by current and former members of Wayfair Engineering.
 
-[https://wayfair.github.io/awesome-learning/](https://wayfair.github.io/awesome-learning/)
+[Check out the site](https://wayfair.github.io/awesome-learning/)
 
 ## Quick Start
 
-#### Start Developing
+#### Working With Our Repo
+1. Fork the Awesome Learning Repo
+2. Clone your fork to your local directory
+3. Change into the local directory
+4. Install dependencies
+```sh
+npm install
+```
+
+#### Starting the Repo
```

Comment:
I'd recommend changing this to `Starting Awesome Learning`. I don't know how much sense `Starting the Repo` makes; it's not just a repo, it's the Awesome Learning platform.